app/scaffold-controllers/navigationController.js

```diff
@@ -11,17 +13,26 @@ function(
     Application,
     AnchoredLayoutPlugin,
     NavigationPlugin,
-    NavigationHeaderController
+    NavigationHeaderController,
+    SearchBarController,
+    SearchConfig
```

Comment:
Thanks for the suggestions. I have done something pretty similar. I have changed it so that you just pass in the full config object to the init function. Let me know what you think?